views/elements/current_and_next.tmpl

```diff
@@ -3,15 +3,16 @@
 
 		<div class="row">
 			{{if .Current}}
-				<div class="col-xs-4 col-sm-4 now-next-live" style="background-image:url('/images/index-now-next-live-thumb.jpeg');"></div>
-				<div class="col-xs-8 col-sm-4 now-next-now">
+				<div id="current-next-live" class="col-xs-4 col-sm-4 current-next-live" style="background-image:url('/images/show-current.png');">
+				<a href="https://ury.org.uk/listen" class="btn btn-lg btn-primary current-next-live-button" title="Listen Live" onclick="window.open('http://ury.org.uk/live', 'radioplayer', 'height=665,width=380'); return false;">Listen Live</a></div>
```

Comment:
One think I've meant to ask, I've noticed that https://ury.org.uk is hard coded in random places around, I would have thought this would be a central config constant set somewhere?